### PR TITLE
Fix link to DDJ 400 manual

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by ronso0** [comment]
LGTM
Though where are the CI checks?

File: source/hardware/controllers/pioneer_ddj_400.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ The Pioneer DDJ-400 is a 2 deck USB controller with an integrated audio interfac
 designed for rekordbox.
 
 - `Manufacturer's Product Page <https://www.pioneerdj.com/en-us/product/controller/ddj-400/black/overview/>`__
-- `Manufacturer's User Manual <http://docs.pioneerdj.com/Manuals/DDJ_400_DRI1551A_manual/>`__
+- `Manufacturer's User Manual <https://support.pioneerdj.com/hc/en-us/articles/4404919716761>`__
```
Comment (ronso0) [inline source/hardware/controllers/pioneer_ddj_400.rst:22]: The links in that page are redundant IMO, all resources are listed in the DDJ400 main support page, except the MIDI messages table and the hardware diagram.

So, let's link to the parent page instead and remove the fixed Firmware link below
```suggestion
- `Manufacturer's Support Page <https://support.pioneerdj.com/hc/en-us/sections/4416577146009-DDJ-400>`__
```

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Comment by ronso0** [comment]
https://deploy-preview-523--mixxx-manual.netlify.app/hardware/controllers/pioneer_ddj_400.html

**Comment by ronso0** [comment]
and `trailing-whitespace` fails

**Comment by JoergAtGithub** [comment]
I think this is superseded by #532 and #529 now?

**Comment by daschuer** [comment]
Oh yes, I forgot about this.

**Pull request closed without merging**